activerecord/lib/active_record/suppressor.rb

```diff
@@ -30,10 +30,11 @@ module Suppressor
 
     module ClassMethods
       def suppress(&block)
+        previous_state = SuppressorRegistry.suppressed[name] || false
```

Comment:
Why the `|| false`. If the previous value was `nil`, we might as well put it back to `nil` no?